pkg/project/cloud/Taskfile.yml

```diff
@@ -149,6 +150,25 @@ tasks:
     cmds:
       - sh aemw package deploy --file "all/target/*.all-*.zip"
 
+  aem:check:
+    deps: [aem:author:check, aem:publish:check]
+
+  aem:author:check:
+    vars:
+      BASIC_AUTH:
+        sh: sh aemw config value -t "[[.Instance.Config.local_author.User]]:[[.Instance.Config.local_author.Password]]" --output-value value 2>&1
```

Comment:
> [[.Instance.Config.local_author.User]]
Why local_author starts with small letter whereas others with capital letter?

Reply:
I don't like it either but the template `[[ ... ]]` reads value from go struct directly, not from yaml representation

Reply:
if I will use .dotenv in task and sync file when `taskw` is executed then all variables from aemc will be available just like that...

`AEM_INSTANCE_CONFIG_LOCAL_AUTHOR_USER` etc

Reply:
done differently